Analytics/Classes/Internal/SEGHTTPClient.h

```diff
@@ -15,6 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SEGHTTPClient : NSObject
 
 @property (nonatomic, strong) SEGRequestFactory requestFactory;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, NSURLSession *> *sessionsByWriteKey;
```

Comment:
While that's true the interface of the SEGHTTPClient is such that we cannot guarantee that only one writeKey is used. I didn't want to make extensive modifications so this is where we ended up.